flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowDataDeserializationSchema.java

```diff
@@ -130,6 +133,39 @@ public RowData deserialize(byte[] message) throws IOException {
 		}
 	}
 
+	@Override
```

Comment:
Remove the implementation of `deserialize(byte[] message)`, because it is never used now.

Reply:
I agree that we should remove this method for json format. However there are many places that still call this method. We can do this work in one separate issue, WDYT?

Reply:
Could you create an issue for this and add comment in the `deserialize(byte[] message)` method? 

I think the problem is that some connector doesn't migrate to `deserialize(byte[] message, Collector<RowData> out)` yet, for example, Kinesis. If we remove the implementation of `deserialize(byte[] message)`, the upcoming Kinesis SQL connector will not support JSON format.

Reply:
I still prefer to only keep one implementation, otherwise it's hard to maintain in the future. We should update the json tests to use collector methods to have full test coverage. 

The Kinesis should migrate to collector method ASAP, rather than hacking JSON format for Kinesis. What do you think?